docs/standard/index.md

```diff
@@ -1,60 +1,60 @@
 ---
-title: .NET Guide
-description: Learn about .NET.
+title: .NET Standard guide
+description: Learn about .NET and .NET Standard.
 author: cartermp
-ms.date: 05/02/2017
+ms.date: 12/27/2019
 ms.technology: dotnet-standard
 ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 ---
-# .NET Guide
+# .NET Standard guide
 
-The .NET Guide provides a large amount of information about .NET.  Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other sections of the .NET documentation.
+The .NET Standard guide provides information about .NET Standard. Depending on your familiarity with .NET, you may wish to explore different sections of this guide and other guides such as [.NET Core](../core/index.md) or [.NET Framework](../framework/index.md).
 
 ## New to .NET
 
-If you want a high-level overview about .NET, check out [What is .NET?](https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet).
+If you want a high-level overview of .NET, see [What is .NET?](https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet).
 
-If you're new to .NET, check out the [Get Started](get-started.md) article.
+If you're new to .NET, see [Get started](get-started.md).
 
-If you prefer to have a guided tour through major features of .NET, check out the [Tour of .NET](tour.md).
+If you prefer to have a guided tour through major features of .NET, see [Tour of .NET](tour.md).
 
-You can also read about [.NET Architectural Components](components.md) to get an overview of the various "pieces" of .NET and how they fit together.
+For an overview of the various "pieces" of .NET and how they fit together, see [.NET architectural components](components.md).
```

Comment:
Is the double quotes necessary here ?

My personal opinions is that they're distracting. The sentence still reads good without it.

```suggestion
For an overview of the various pieces of .NET and how they fit together, see [.NET architectural components](components.md).
```